### DISCUSSION TITLE
Import only normal comments from Disqus

### DISCUSSION THREAD

**Discussion by Ivan-Chupin** [umputun/remark42#988] [Ideas]
Import from Disqus adds absolutely all comments to Remark42, including deleted and spam comments. I believe that you only need to import comments that are not marked as deleted and spam.

**Comment by umputun** [comment]
this is not entirely accurate. Comments reported in the export as spam should be ignored https://github.com/umputun/remark42/blob/master/backend/app/migrator/disqus.go#L123

**Reply by umputun** [reply]
not sure about deleted comments, are they even exported by disquss?

**Reply by Ivan-Chupin** [reply]
Deleted comments are exported to an XML file.

**Reply by Ivan-Chupin** [reply]
I checked. Indeed, comments marked as spam are not added to the database, unlike deleted ones.

**Reply by umputun** [reply]
ok, this is a bug for sure. will deal with this. As I see deleted can be on a comment level as well as on the thread level

**Comment by umputun** [comment]
the change/fix in the master- https://github.com/umputun/remark42/pull/989
